package.json

```diff
@@ -94,26 +86,27 @@
     "electron-icon-maker": "^0.0.4",
     "esm": "^3.2.25",
     "fancy-log": "^1.3.3",
+    "globby": "^13.1.1",
     "gulp": "^4.0.0",
     "html-webpack-plugin": "^5.1.0",
     "husky": "^1.3.1",
+    "intl-messageformat": "^9.12.0",
     "karma": "^6.3.16",
     "karma-chrome-launcher": "^3.1.0",
     "karma-jasmine": "^4.0.1",
     "karma-webpack": "^5.0.0",
+    "lit": "^2.2.2",
```

Comment:
Why is this moving to dev? It's a direct dependency of our web code.

Reply:
At this moment we're only using it in storybook.